lib/etl/jobs/user_match_runner.ts

```diff
@@ -1,46 +1,70 @@
-import {runBackfills, runRefresh, runUpdates} from '../user_match_scrape';
-import { createEtlJob, updateEtlJob } from '../../model/etl_jobs';
-import { argv } from 'yargs';
+import {argv} from 'yargs';
 import md5 from 'md5';
 
-const jobType: string = argv.jobType as string;
+import {runBackfills, runRefresh, runUpdates} from '../user_match_scrape';
+import {createEtlJob, updateEtlJob} from '../../model/etl_jobs';
 
-const validJobTypes: Array<string> = ['update','backfill'];
-if (!jobType || !validJobTypes.includes(jobType)) {
-    throw new Error('option jobType is required and needs to be in ' + validJobTypes.join(','));
-}
-const jobNames = {
-    update: 'user_matches_update',
-    backfill: 'user_matches_backfill'
-};
-const jobTime: number = new Date().getTime();
-const newJob = {
-    job_name: jobNames[jobType],
-    job_id: md5(jobTime + jobNames[jobType]),
-    execution_start_time: jobTime,
-    is_successful: null,
-    execution_end_time: null
+const CONFIG = {
+    JOBS: {
+        update: {
+            name: 'user_matches_update',
+            method: runUpdates
+        },
+        backfill: {
+            name: 'user_matches_backfill',
+            method: runBackfills
+        },
+        refresh: {
+            name: 'user_matches_refresh',
+            method: runRefresh
+        }
+    }
 };
 
-const jobFunctions = {
-    update: runUpdates,
-    backfill: runBackfills,
-    refresh: runRefresh
-};
-async function run() {
-    const jobFunction = jobFunctions[jobType];
+const VALID_JOB_KEYS = Object.keys(CONFIG.JOBS);
```

Comment:
Object.keys to validate the valid jobs instead of explicitly writing the different job names in multiple places like before (which is what caused the refresh to not run)